circle.yml

```diff
@@ -35,6 +35,7 @@ dependencies:
   cache_directories:
     - ~/sauce-connect
   pre:
+    - sudo apt-get update; sudo apt-get install imagemagick
```

Comment:
maybe comment mentioning that this is for the `convert` tool that's used later? Otherwise it's confusing since the name `imagemagick` never shows up again